Source/cmake/OptionsWPE.cmake

```diff
@@ -119,6 +119,7 @@ if (ENABLE_DEVELOPER_MODE)
     WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_API_TESTS PRIVATE ON)
     WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_MINIBROWSER PUBLIC ON)
     WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_COG PRIVATE ON)
+    WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_BREAKPAD PUBLIC ON)
```

Comment:
Adrian, can you confirm you want this ON by default?

Reply:
Ahhh, good catch, I would actually prefer that this is `OFF` by default, to not add a new dependency which is not always packaged in distros (and whoever needs Breakpad support in an embeded device _should_ know what they are up to and how to enable it.)